File: app/src/main/java/com/song/deviceinfo/ui/hardware/HardwareViewModel.java
package com.song.deviceinfo.ui.hardware;

import android.content.Context;

import com.song.deviceinfo.info.HardwareInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2021/9/29.
 */
public class HardwareViewModel extends NormalViewModel {

    public List<Pair<String, String>> getHardwareInfo(Context context) {
        return HardwareInfo.getHardwareInfo(context);
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/hardware/HardwareViewModel.kt
package com.song.deviceinfo.ui.hardware

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.HardwareInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2021/9/29.
 */
class HardwareViewModel : NormalViewModel() {
    fun getHardwareInfo(context: Context): List<Pair<String, String>> {
        return HardwareInfo.getHardwareInfo(context)
    }
}
